views/movies/movie-edit.hbs

```diff
@@ -0,0 +1,25 @@
+<h2>Edit the movie</h2>
+
+<form action="/movies/create" method="post">
```

Comment:
When you edit, you are sending information to the create route -> so every time I try toe dit movie I am actually creating one.

Here you should be using the edit URL that you defined in movie routes. 
```
<form action="/movies/{{id}}/edit" method="post">
```

Reply:
Thank you! I couldn't figure out why it didn't work.